doc/examples/sample_repos.md

```diff
@@ -256,43 +256,75 @@ index.ipynb
 
 ### JupyterLab + Binder
 
-[![Binder](http://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/binder-examples/jupyterlab/master?urlpath=lab/tree/index.ipynb) | [![](https://img.shields.io/github/forks/binder-examples/jupyterlab?label=GitHub%20Repo&style=social)](https://github.com/binder-examples/jupyterlab)
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/binder-examples/jupyterlab/HEAD?filepath=/index.ipynb) | [![](https://img.shields.io/github/forks/binder-examples/jupyterlab?label=GitHub%20Repo&style=social)](https://github.com/binder-examples/jupyterlab)
+
+JupyterLab is the default UI on Binder.
+
+#### Create a Binder link that points to a file in JupyterLab
+
+You can also create a Binder link that points to a specific file in JupyterLab by adding the following
+to the end of your link:
+
+```
+?filepath=path/to/my/notebook.ipynb
+```
+
+
+For example, the Binder badge above goes to the following URL:
+
+`https://mybinder.org/v2/gh/binder-examples/jupyterlab/HEAD?filepath=/index.ipynb`
+
+Note: this repository also installs several JupyterLab extensions via a `postBuild` script, allowing
+you to use JupyterLab's extensions and widgets functionality.
+
+For a more complete demo of JupyterLab using Binder, see the
+[JupyterLab Demo](https://github.com/jupyterlab/jupyterlab-demo).
+
 
-JupyterLab is packaged with Binder repositories by default. In order to
-run a JupyterLab session, you have two options:
+### Classic notebook interface on Binder
 
-#### Start JupyterLab after you start your Binder
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/binder-examples/requirements/HEAD?urlpath=/notbooks/index.ipynb) | [![](https://img.shields.io/github/forks/binder-examples/requirements?label=GitHub%20Repo&style=social)](https://github.com/binder-examples/requirements)
+
+JupyterLab is the default UI on Binder,
+but the classic notebook is available, too.
+In order to run a JupyterLab session, you have two options:
+
+#### Start classic notebook after you start your Binder
 
 Do the following:
 
 1. Launch a Binder instance (e.g., by clicking the Binder badge)
-2. Replace `tree` at the end of your URL with `lab`.
+2. Replace `lab` at the end of your URL with `tree`.
 3. That's it!
 
-#### Create a Binder link that points to JupyterLab
+#### Create a Binder link that points to classic notebook
 
-You can also create a Binder link that points to JupyterLab by adding the following
-to the end of your link:
+You can also create a Binder link that points to the classic notebook landing page
+by adding the following to the end of your link:
 
-`?urlpath=lab`
+```
+?urlpath=/tree/
+```
 
-You can point to a specific file using JupterLab by including a file path
-beginning with `tree/` to the end of `urlpath`, like so:
+You can point to a specific file using the classic notebook interface by including a file path
+beginning with `/tree/` to the end of `urlpath`, like so:
```

Comment:
In the below examples we don't start with `/tree` and https://mybinder.org/v2/gh/binder-examples/requirements/HEAD?urlpath=/notebooks/index.ipynb works. So I think we should remove the last part of the sentence

Reply:
That's confusing to me - I'd assume that of we specified urlpath without "tree" that it would direct to a lab UI now...is that not right?

Reply:
This is an implementation detail of the notebook server that users shouldn't need to know about, but:

`/tree/` and `/notebooks/` URLs are equivalent - if you link to `/tree/` but it finds a file, it will redirect you to `/notebooks`/ or `/view/` etc. as appropriate, based on file type. `/notebooks/` does the same in the other direction.

So I see two main choices:

1. always use `/tree/` because it always works and is less for everyone to think about (though it might be confusing for folks who look at the URL in their address bar and see `/notebooks/` instead of `/tree/`), or
2. put the *actual* destination (`/notebooks/` for a notebook and `/tree/` for a directory), and explain a bit about "copy the URL from your browser" and different endpoints in the docs here.

We have to do 1. in our internal redirect internally, because BinderHub can't actually know what you are linking to when it makes the URL.

My inclination, given our audience, is to just use `/tree/` in the URLs and not get too tangled in the weeds of how it works.

Reply:
This PR now reflects the `/tree/` everywhere case, lmk if you think it should be otherwise